lib/shardingcfg.go

```diff
@@ -300,7 +305,7 @@ func InitShardingCfg() error {
 				}
 				db, err = openDb(shard)
 				if err == nil {
-					err = loadMap(ctx, db)
+					err = loadMap(&ctx, db, reloadInterval/2)
```

Comment:
Same as above. Let us use a configurable query timeout for maintenance queries

Reply:
Done